.github/workflows/algebra-extension.yml

```
@@ -34,3 +35,9 @@ jobs:
         working-directory: extensions/algebra/circuit
         run: |
           cargo nextest run --cargo-profile=fast
+
+      - name: Run algebra integration tests
+        working-directory: extensions/algebra/tests
+        run: |
+          rustup component add rust-src --toolchain nightly-2024-10-30-aarch64-unknown-linux-gnu
```

```suggestion
          rustup component add rust-src --toolchain nightly-2024-10-30
```
apparently just this works and is not architecture specific